vsphere_plugin_common/__init__.py

```diff
@@ -409,6 +410,59 @@ def _get_datastores(self, use_cache=True):
             use_cache=use_cache,
         )
 
+    def _get_connected_network_name(self, network):
+        name = None
+        if network.get('from_relationship', False):
```

Comment:
It's nice to be explicit about these things though- though yes, in this specific case it doesn't make any difference except to be clear about what we're expecting from the from_relationship. Worth changing?